src/ds3.h

```diff
@@ -357,6 +356,10 @@ LIBRARY_API ds3_creds*  ds3_create_creds(const char* access_id, const char* secr
 LIBRARY_API ds3_client* ds3_create_client(const char* endpoint, ds3_creds* creds);
 LIBRARY_API ds3_error*  ds3_create_client_from_env(ds3_client** client);
 LIBRARY_API void        ds3_client_register_logging(ds3_client* client, ds3_log_lvl log_lvl, void (* log_callback)(const char* log_message, void* user_data), void* user_data);
+LIBRARY_API void        LOG(const ds3_log* log, ds3_log_lvl lvl, const char* message, ...);
```

Comment:
Put this into an internal util header or something like that.  We shouldn't publish this outside of our code.  Same comment for `load_buffer` and `ds3_create_error`